src/session.js

```diff
@@ -0,0 +1,22 @@
+function methods(request, opt) {
+  return {
+    state: null,
+
+    get() {
+      return request('get', '/session');
+    },
+
+    getCookie() {
+      this.state = opt.getCookie('swell-session');
+      return this.state;
+    },
+
+    setCookie(value) {
+      this.state = value;
```

Comment:
Maybe set `this.state` after setting the cookie

Reply:
Removed the property, not really needed